### DISCUSSION TITLE
Live timing saved layout outside of an active ‘live’ session

### DISCUSSION THREAD

**Discussion by basketballhead6** [f1multiviewer/issue-tracker#448] [Feature requests]
I can only use F1 Access and use MV for the enchanted live timing.

Would it be possible if you open the live timing window that you can open a saved layout? I've saved my live timing window as specific layouts. This way I can open my live timing screens even through a session is not marked 'live'.

**Comment by LapsTimeOFF** [comment]
This has been implemented since multiple version, closing.
